ml-agents/mlagents/trainers/env_manager.py

```diff
@@ -67,6 +67,10 @@ def reset(self, config: Dict = None) -> int:
         self.first_step_infos = self._reset_env(config)
         return len(self.first_step_infos)
 
+    @abstractmethod
+    def send_to_environment(self, config: Dict = None) -> None:
```

Comment:
Might want to rename to `set_env_parameters` or something similar to be more descriptive. Or add a good docstring

Reply:
thats a better name